src/main/g8/src/main/java/com/example/helloworld/GreeterServer.java

```diff
@@ -60,8 +60,9 @@ public GreeterServer(ActorSystem system) {
     CompletionStage<ServerBinding> bound =
         Http.get(system).bindAndHandleAsync(
           service,
-          ConnectWithHttps.toHostHttps("127.0.0.1", 8080)
-              .withCustomHttpsContext(serverHttpContext()),
+          ConnectHttp.toHost("127.0.0.1", 8080),
+          //ConnectWithHttps.toHostHttps("127.0.0.1", 8080)
+          //    .withCustomHttpsContext(serverHttpContext()),
```

Comment:
It seems HTTPS worked before fairly recently (https://github.com/akka/akka-grpc-quickstart-java.g8/pulls?q=is%3Apr+is%3Aclosed), let's fix that when master is back to green though